### ISSUE TITLE
Analyzing anchors... Illegal instruction (core dumped)

### ISSUE THREAD

**Issue by MattVil** [ultralytics/yolov5#1224]
Hey, I tried to train a Yolo model on the [EgoHands](http://vision.soic.indiana.edu/projects/egohands/) dataset following your [Train Custom Data](https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data) Tutorial but when i'm running :  
`python3 train.py --img 640 --batch 4 --epochs 30 --data ./data/hands.yaml --cfg ./models/yolov5s.yaml --weights weights/yolov5s.pt --name yolov5s_hands --cache`

I got the following error no matter what Yolo model I use : 
```
...
Caching images (3.0GB): 100%|████████████████████████████████████| 4400/4400 [01:13<00:00, 59.96it/s]
Scanning labels ../EgoHands/labels/test.cache (399 found, 0 missing, 1 empty, 0 duplicate, for 400 images): 400it [00:00, 4303.81it/s]
Caching images (0.3GB): 100%|██████████████████████████████████████| 400/400 [00:06<00:00, 58.31it/s]

Analyzing anchors... Illegal instruction (core dumped)
```

In the model config file I just changed the number of categories and the dataset config file look like that :
```
train: ../EgoHands/images/train/
val: ../EgoHands/images/test/
nc: 4
names: ["myright", "myleft", "yourright", "yourleft"]
```
My environment settings are:
OS: Ubuntu 18
GPU: GTX1080ti
Python: 3.6.9
Torch: 1.6.0
Cuda: 10.2

**Comment by MattVil** [comment]
Fixed by upgrading torch to 1.7.0

**Comment by glenn-jocher** [comment]
@MattVil that's interesting, thanks for the feedback!

You can also skip this autoanchor step with `python train.py --noautoanchor`. You can also omit the --cfg argument completely when loading from pretrained --weights, as it will inherit all the architecture it needs to from the --weights.
